content/contribute/elasticsearch-develop.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "ElasticSearch notifier development"
+weight: 11
+---
+
+To develop BotKube with Elasticsearch notifier enabled, install it locally:
+
```

Comment:
```suggestion

### Steps

```

To be consistent with the already existing doc: https://github.com/infracloudio/botkube-docs/blob/master/content/contribute/ms-teams-develop.md